raster-foundry/raster-foundry
# Refactor Sharing view to use standard markup

## Overview

Refactor Sharing view to use classes from Scenes view, which are the standard markup for these types of components.

### Demo

![image](https://user-images.githubusercontent.com/1809908/32291867-bcdb4c4a-bf14-11e7-9320-ebd7b60f968c.png)

### Comments

I am using `.list-group-item` and `.sidebar-actions-group` because both seem appropriate. These classes are used in `scenes.html`, though on different elements. I am using them on the same element.

## Testing Instructions

 * Open a project
 * Switch to Sharing view
 * Make sure the action buttons in the sidebar look good
 * Review markup to make sure proper classes are used
